common/pom.xml

```diff
@@ -105,10 +105,12 @@
     <dependency>
       <groupId>org.pentaho</groupId>
       <artifactId>pentaho-hadoop-shims-api</artifactId>
+      <version>${project.version}</version>
```

Comment:
Yes you're right. I must have done this by mistake looking for the other pentaho version thing (I wasn't up to date with your changes).